jwst/associations/tests/make_headers.py

```diff
@@ -25,7 +25,7 @@ def actual_path(path):
     Path
         The fully qualified path.
     """
-    return Path.abspath(ospath.expandvars(Path.expanduser(path)))
+    return Path(ospath.expandvars(Path(path).expanduser())).resolve()
```

Comment:
No need to contruct Path twice? Please do double check.
```suggestion
    return Path(ospath.expandvars(path)).expanduser().resolve()
```